lib/Raven/ErrorHandler.php

```diff
@@ -140,6 +140,13 @@ public function handleFatalError()
 
     public function shouldCaptureFatalError($type)
     {
+        // Do not capture E_ERROR since those can be caught by userland since PHP 7.0
+        // E_ERROR should already be handled by the exception handler
+        // This prevents duplicated exceptions in PHP 7.0+
+        if (version_compare(phpversion(), '7.0', '>=') && $type === E_ERROR) {
```

Comment:
Maybe `if (PHP_VERSION_ID >= 70200)` is faster? That constant is available since PHP 5.2, so that's not an issue to use it.

Reply:
Looks like my PhpStorm exceptions thought the same way. I changed it everywhere to stay consistent.